src/main/resources/templates/repo/sns-and-sqs-config.json

```diff
@@ -155,7 +155,8 @@
 			"subscribedTopicNames":[
 				"TABLE"
 			],
-			"messageVisibilityTimeoutSec":1200
+			"messageVisibilityTimeoutSec":1200,
+			"messageRetentionPeriodSec":1209600
```

Comment:
Maybe add the MATERIALIZED_VIEW_UPDATE, MATERIALIZED_VIEW_SOURCE_UPDATE and SEARCH_UPDATE queues as well?